pkg/provider/provider.go

```diff
@@ -36,8 +46,35 @@ func Bootstrap(manifestPath string, force bool) (Provider, error) {
 	return Select(force)
 }
 
+func GetProviderScaffold(provider string) (string, error) {
```

Comment:
I think these functions should probably be private not public, but not a huge deal